src/pkg/services/m365/sites_test.go

```diff
@@ -36,16 +38,15 @@ func TestSiteIntegrationSuite(t *testing.T) {
 
 func (suite *siteIntegrationSuite) SetupSuite() {
 	t := suite.T()
+	var err error
```

Comment:
same here, could you please move this closer to the actual usage of the variable?